COLLABORATOR_GUIDE.md

```diff
@@ -87,6 +87,28 @@ All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
 
+Here are some useful CI jobs:
+
+* [`node-test-pull-request`](https://ci.nodejs.org/job/node-test-pull-request/)
+is the standard CI run we do to check Pull Requests. It triggers `node-test-commit`, 
+which runs the `build-ci` and `test-ci` targets on all supported platforms.
+
+* [`node-test-linter`](https://ci.nodejs.org/job/node-test-linter/)
+only runs the linter targets, which is useful for changes that only affect comments 
+or documentation.
+
+* [`citgm-smoker`](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/)
```

Comment:
This could be the shorter `https://ci.nodejs.org/job/citgm-smoker/` if you omit the `view/Node.js-citgm`. Same for `citgm-abi-smoker` later in this paragraph.